app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,32 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assignedContact: null,
+  tags: null,
 };
 
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
+  assignedContact: yup
+    .object()
+    .shape({
+      label: yup.string().required(),
+      value: yup.string().required(),
+    })
+    .nullable()
+    .required("Assigned contact is required"),
+  tags: yup
+    .array()
+    .of(
+      yup.object().shape({
+        label: yup.string(),
+        value: yup.string(),
+      })
+    )
+    .nullable()
```

Comment:
I believe this nullable is added over here since you set up the initial value as null. If we change it to [] then I don't think nullable is needed here.